src/helm/clients/vision_language/huggingface_vlm_client.py

```diff
@@ -90,11 +94,14 @@ def do_it() -> Dict[str, Any]:
         except RuntimeError as e:
             return RequestResult(success=False, cached=False, error=str(e), completions=[], embedding=[])
 
+        output: str = result["generated_text"]
+        if "ASSISTANT: " in output:
```

Comment:
Is this token model-specific?

Reply:
I think it's general for the chat models.